pkg/kubelet/cm/deviceplugin/types.go

```diff
@@ -28,8 +31,8 @@ type MonitorCallback func(resourceName string, added, updated, deleted []plugina
 
```

Comment:
Does MonitorCallback still need to be public?

Reply:
Trying to revert to a private one now